### DISCUSSION TITLE
Crash at startup

### DISCUSSION THREAD

**Discussion by LeoFlash13** [ChefKissInc/NootedRed#93] [Q&A] [answered]
Hi all, I have a laptop with a Ryzen 2200u and an integrated Vega 3. Mac OS Monterey starts without any problem without video kext but if I enter Nootedred it gives the error it shows in the picture, does anyone know how to help me?
![IMG-20230613-WA0003](https://github.com/NootInc/NootedRed/assets/98963936/95a21c17-a46b-459d-8285-0e8c1cc0148b)

**Comment by RubRLO** [comment]
![photo_4931762980534725425_y](https://github.com/NootInc/NootedRed/assets/59743626/faef9f3c-d2c6-4656-93f5-ea2f52031c5d)
![photo_4931962894082485610_y](https://github.com/NootInc/NootedRed/assets/59743626/6281d67a-954c-4caf-b4b6-18e165051171)

If I'm getting it, I would like some help to get past this error. It already recognized my computer and processor when receiving the VRAM

**Comment by VisualEhrmanntraut** [comment, marked as answer]
See README and pinned discussion
